<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverdatadialog</class>
 <widget class="QDialog" name="serverdatadialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>524</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit server</string>
  </property>
  <property name="windowIcon">
   <iconset resource="stubby.qrc">
    <normaloff>:/images/stubby.ico</normaloff>:/images/stubby.ico</iconset>
  </property>
  <property name="toolTip">
   <string extracomment="Edit server properties"/>
  </property>
  <property name="toolTipDuration">
   <number>-7</number>
  </property>
  <property name="statusTip">
   <string extracomment="B"/>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>461</width>
     <height>164</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="serverName">
      <property name="text">
       <string>Name*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Website</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="toolTip">
       <string extracomment="Edit the IP Address of a server (only IPv4 accepted)"/>
      </property>
      <property name="statusTip">
       <string extracomment="A"/>
      </property>
      <property name="text">
       <string>IP Address*</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>TLS Auth Name</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Pin</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="serverNameData"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="serverWebsiteData"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="serverAddressData"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="serverAuthName"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="serverPinData"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="v6AddressLabel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Main IPv6 Address</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="v6AddressEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="validationMessage">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="discardButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard Edits</string>
   </property>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="stubby.qrc"/>
 </resources>
 <connections/>
</ui>
